Add changes related to FTL#1409 (#774)
pi-hole/docs: docs/database/ftl.md

#### Data-dependent `additional_info` field

The content and type of the `additional_info` row depends on the status of the given query. For many queries, this field is empty. You should, however, not rely on this field being empty as we may add content of any type for other status types in future releases.

##### Query blocked due to a CNAME inspection (status 9, 10, 11) {#additional_info_cname data-toc-label='Blocked CNAME'}

If a query was blocked due to a CNAME inspection (status 9, 10, 11), this field contains the domain which was the reason for blocking the entire CNAME chain (text).

> ##### Query blocked due to a black- or whitelist entry {#additional_info_list data-toc-label='Regular expression'}

Reviewer: How can a whitelist entry block something?

> ##### Query blocked due to a black- or whitelist entry {#additional_info_list data-toc-label='Regular expression'}

Reviewer: ```suggestion
##### Query blocked due to a black- or whitelist entry {#additional_info_list data-toc-label='Domainlist domain'}
```

> ##### Query blocked due to a black- or whitelist entry {#additional_info_list data-toc-label='Regular expression'}

Author: ```suggestion
##### Query influenced by a black- or whitelist entry {#additional_info_list data-toc-label='domainlist_id'}
```
That's the best idea I can come up with right now.

> If a query was blocked due to a black- or whitelist entry, this field contains the ID of the corresponding entry in the [`domainlist`](gravity/index.md#domain-tables-domainlist) table.

Reviewer: ```suggestion
If a query was influenced due to a black- or whitelist entry, this field contains the ID of the corresponding entry in the [`domainlist`](gravity/index.md#domain-tables-domainlist) table.
```
